assets/src/settings-page/plugin-suppression.js

```diff
@@ -292,11 +292,7 @@ PluginRow.propTypes = {
  * Component rendering the plugin suppression table.
  */
 export function PluginSuppression() {
-	const { editedOptions, fetchingOptions } = useContext( Options );
-
-	if ( fetchingOptions ) {
-		return null;
```

Comment:
We don't need `fetchingOptions` in `supported-templates.js` either now I presume.

Reply:
Removed in 35e5fbe5d